src/Microsoft.DotNet.Web.Spa.ProjectTemplates/content/React-CSharp/ClientApp/src/App.js

```diff
@@ -6,6 +6,7 @@ import { FetchData } from './components/FetchData';
 import { Counter } from './components/Counter';
 
 export default class App extends Component {
+  static displayName = App.name
```

Comment:
Ultra-minor nitpick: You have a blank line after the equivalent to this line in other files.